docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+
+:::note
+Note the `global.multiregion.regions` directive indicates the use for two regions. Disable Identity and Optimize as these are currently not supported within this setup and will break when enabled. Please see the [limitations section](#) <!-- TODO: fill link -->.
+:::
+
+##### region0/camunda-values.yml
+
+This overlay contains the multi region identification for the cluster in region 0.
+
+##### region1/camunda-values.yml
+
+This overlay contains the multi region identification for the cluster in region 1.
+
+#### Preparation
+
+The base `camunda-values.yml` requires some adjustments before installing the Helm chart.
+
+1. Adjust the `placeholder` for `ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS` to reflect your current setup. These are the contact points for the brokers to know how to form the cluster. Find more information on what the variable means in [setting up a cluster](../../../../zeebe-deployment/operations/setting-up-a-cluster.md).
+
+<!-- TODO: currently on a different branch - infex-98-docs-changes -->
+
+**Use the python script in the root of the repository `generate_zeebe_initial_contact.py` to generate the replacement string.**
+
+```bash
+python3 generate_zeebe_initial_contact.py
+
+# It will ask you to provide the following values
+# Enter a namespace for cluster in region 0:
+## in our case camunda-london
+# Enter a namespace for cluster in region 1:
+## in our case camunda-paris
+# Enter Helm release name:
+## the way you'll call the Helm release, for example camunda
+# Enter Zeebe cluster size:
+## for a dual-region setup we recommend 8
+```
+
+In our example, the `placeholder` is replaced with the following:
+
+```yaml
+camunda-zeebe-0.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-london.svc.cluster.local:26502,camunda-zeebe-0.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-1.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-2.camunda-zeebe.camunda-paris.svc.cluster.local:26502,camunda-zeebe-3.camunda-zeebe.camunda-paris.svc.cluster.local:26502
+```
+
+The Zeebe cluster size is eight in a dual-region setup, resulting in four brokers per region.
+
+2. In case the Helm release name differs from `camunda` and your chosen namespaces, you must adjust the `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION0_ARGS_URL` and `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION1_ARGS_URL` values.
+
+They are constructed as follows:
+
+```bash
+http://${HELM_RELASE_NAME}-elasticsearch-master-hl.${CAMUNDA_NAMESPACE}.svc.cluster.local:9200
+# Where HELM_RELEASE_NAME is your chosen helm release name
+# and CAMUNDA_NAMESPACE is your previously defined CAMUNDA_NAMESPACE_0 and CAMUNDA_NAMESPACE_1, depending on ELASTICSEARCHREGION0 or ELASTICSEARCHREGION1
```

Comment:
I'm confused about this part, what does `depending on ELASTICSEARCHREGION0 or ELASTICSEARCHREGION1` mean?

`ELASTICSEARCHREGION0` and `ELASTICSEARCHREGION1` have not been previously mentioned.

Reply:
rephrased it and added an example.